092-self-healing-using-cruise-control.md

```diff
@@ -177,13 +183,13 @@ The event published would look like this:
   type: Normal
 ```
```

Comment:
https://github.com/strimzi/proposals/pull/152 - would this proposal have an impact on the format of the Kubernetes event for consistency?

Reply:
Not sure, the notifier would reside as a separate repo under Strimzi, but I can adapt the events to look like this in the proposal. Wdyt @ppatierno?

Reply:
Yes we should aim to have consistency across what we'll propose here and what's proposed in #152